friendly_traceback/path_info.py

```diff
@@ -55,19 +64,19 @@ def exclude_directory_from_traceback(dir_name):
 exclude_directory_from_traceback(dirname)
 
 
-def is_excluded_file(full_path):
+def is_excluded_file(full_path: StrPath) -> bool:
     """Determines if the file belongs to the group that is excluded from tracebacks."""
-    if full_path.startswith("<frozen "):
```

Comment:
@aroberge I have allowed me to fix this small issue along the way - if `full_path` is any `PathLike` (`pathlib.Path` etc), the check will fail since `startswith` is not part of the file system path protocol. Ensuring `full_path` is converted to a string by swapping both statements is an easy fix. Do you want me to add a test for the fix?

Reply:
@hoefling If  you wouldn't mind adding a test, that would be great indeed.